### DISCUSSION TITLE
能够添加批量翻译本地字幕文件吗，一个一个上传下载太痛苦了

### DISCUSSION THREAD

**Discussion by zd616nqs** [immersive-translate/immersive-translate#574] [Ideas]
### 新功能 | New Feature

如题，希望能够批量翻译字幕，看教程的字幕文件经常几十个上百个，一个一个上传再下载太痛苦了

### 为什么您想要添加这些功能 | Why you want to add these features

如题

**Comment by despairTK** [comment]
请新增这个功能！
